dasharo/project_config.yaml

```diff
@@ -0,0 +1,36 @@
+# Dasharo project configuration data
+
+dasharo:
+  platform: msi_ms7d25
```

Comment:
@miczyg1 please use the latest file format (e.g. the newsletter url link)

Reply:
Yes, I haven't filled all the necessary information back then. Now it is fixed. asciinema still missing.